### DISCUSSION TITLE
Revised Forward in documentation (PDF, HTML, adoc)

### DISCUSSION THREAD

**Discussion by allthewayman** [usnistgov/macos_security#82] [Ideas]
Can we add more clarification in the Foreword. Updated text proposed below:

1. Foreword

Document Description: 

This document records the settings to achieve this security configuration baseline, as named on the cover page.

To achieve compliance with this baseline, apply the settings as prescribed herein.

Intended Audiences:

System Administrators, Information Security Officers, Compliance Officers, Audit and Business Assurance teams, and Risk Management teams.

Project Description:

The macOS Security Compliance Project is an open source effort to provide a programmatic approach to generating security guidance. The configuration settings in this document were derived from National Institute of Standards and Technology (NIST) Special Publication (SP) 800-53, Security and Privacy Controls for Information Systems and Organizations, Revision 5.

This project can be used as a resource to easily create customized security baselines of technical security controls by leveraging a library of atomic actions which are mapped to the compliance requirements defined in NIST SP 800-53 (Rev. 5). It can also be used to develop customized guidance to meet the particular cybersecurity needs of any organization.

Project Goal:

The objective of this effort was to simplify and radically accelerate the process of producing up-to-date macOS security guidance that is also accessible to any organization and tailorable to meet each organization’s specific security needs.

**Comment by robertgendler** [comment]
We're working on adding a Scope section to the documents generated. This would be based off of text provided in the description tag of the baseline yaml file. So that may help with adding some clarification as that can be used to really describe what the description of this specific baseline is. 

This can be tested out in the cis branch currently.

**Reply by allthewayman** [reply]
That's helpful. The primary goal is to describe the purpose of the document.
